lib/KTooltip/index.vue

```diff
@@ -64,6 +68,13 @@
         type: String,
         default: 'bottom',
       },
+      /**
+       * Text of the tooltip
+       */
+      text: {
+        type: String,
+        default: '',
```

Comment:
Setting the default to an empty string should generally be avoided. If the prop is not required, it is more maintainable to set the default to `null` instead.